controllers/machinedeployment_sync_test.go

```diff
@@ -397,3 +398,105 @@ func TestScaleMachineSet(t *testing.T) {
 		})
 	}
 }
+
+func newDeploymentStatus(replicas, updatedReplicas, availableReplicas int32) clusterv1.MachineDeploymentStatus {
```

Comment:
what is this func giving us over just inlining in the one below?

Reply:
removed it.